CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+Contributing
+============
+
+Thanks for taking the time to contribute to Orange!
+
+Please submit contributions in accordance with the flow explained in the
+[GitHub Guides].
+
+[GitHub Guides]: https://guides.github.com/
+
+
+Reporting bugs
+--------------
+Please report bugs according to established [bug reporting guidelines].
+
+[bug reporting guidelines]: https://www.google.com/search?q=reporting+bugs
+
+
+Coding style
+------------
+
+Roughly conform to [PEP-8] style guide for Python code. Whenever PEP-8 is
+undefined, adhere to [Google Python Style Guide].
+
+In addition, we add the following guidelines:
+* Never `import *`.
```

Comment:
Except when stuff is pulled up into a parent package?

https://github.com/biolab/orange3/blob/master/Orange/classification/__init__.py